wolfcrypt/src/fe_low_mem.c

```diff
@@ -55,15 +55,17 @@ void lm_copy(byte* x, const byte* a)
         x[i] = a[i];
 }
 
+#ifndef FREESCALE_LTC_ECC
 #if ((defined(HAVE_CURVE25519) && !defined(CURVE25519_SMALL)) || \
-	 (defined(HAVE_ED25519) && !defined(ED25519_SMALL))) &&      \
-	!defined(FREESCALE_LTC_ECC)
-	/* to be Complementary to fe_operations.c */
+    (defined(HAVE_ED25519) && !defined(ED25519_SMALL))) &&      \
+    !defined(FREESCALE_LTC_ECC)
+    /* to be Complementary to fe_low_mem.c */
```

Comment:
This section of code should match what is in master I believe.  Kojo recently changed this section.  Please review.

Reply:
I modified it to match master branch code.